back/config/schemas/settings.schema.json.erb

```diff
@@ -914,8 +914,8 @@
         "additionalProperties": false,
         "required": ["allowed", "enabled"],
         "properties": {
-          "allowed": { "type": "boolean", "default": true },
-          "enabled": { "type": "boolean", "default": false }
+          "allowed": { "type": "boolean", "default": false },
+          "enabled": { "type": "boolean", "default": true }
```

Comment:
Because I have now set `allowed` & `enabled` values for this feature in 3 pricing plans that previously had no values for it, this default is not very important.

Since 5 of 6 pricing plans have `allowed: false`, however, this default seems more logical.